### PR TITLE
Fixed: CPViewAnimator was not giving correct frame information

### PR DESCRIPTION
**Pull request by didierkorthoudt**
First work

### CODE REVIEW

**Comment by didierkorthoudt** [comment]
Now, correct frame is computed during the view animation.
@cacaodev : as you will see, I've tried to make this working also for frameOrigin but without success. Any idea ? Also, a final setFrame (or setFrameOrigin, or setFrameSize according to animation going on) should be fired without the _setInhibitDOMUpdate but I didn't find where until now.

File: AppKit/CoreAnimation/CPAnimationContext.j [hunk 1]
```diff
@@ -677,31 +677,44 @@ FrameUpdater.prototype.addTarget = function(target, keyPath, duration)
...
                 return ROUND(parseFloat(style.getPropertyValue(prop)));
-          };
+          },
 
-    var updateFrame = function(timestamp)
+        initialFrame  = [aView frame],
+        initialX      = initialFrame.origin.x,
+        initialY      = initialFrame.origin.y,
+        initialWidth  = initialFrame.size.width,
+        initialHeight = initialFrame.size.height,
```
Comment (Dogild) [inline AppKit/CoreAnimation/CPAnimationContext.j]: Just keep the initialFrame. Make a copy of it, so it will not be updated with the animation

**Comment by cacaodev** [comment]
Bravo for getting the frame values from the matrix, the toughest part. 
I will have a look to the frameOrigin: not getting called later. About the final step: for all animations, there is an internal completion handler (CompletionHandlerAgent) that syncs css values and cappuccino ivars by calling the cappuccino method related to the keyPath. If I remember well (not tested) it was disabled for animations getting frame updates because redundant.

**Comment by didierkorthoudt** [comment]
Replaced by #2502

**Pull request closed without merging**